storage/innobase/fsp/fsp0fsp.cc

```diff
@@ -895,15 +912,20 @@ fsp_header_fill_encryption_info(
 	byte			key_info[ENCRYPTION_KEY_LEN * 2];
 	ulint			crc;
 	Encryption::Version	version;
-#ifdef	UNIV_ENCRYPT_DEBUG
-	const byte*		data;
-	ulint			i;
-#endif
 
+	if(is_boot) {
+		master_key_id = 0;
+		master_key = static_cast<byte*>(ut_zalloc_nokey(
```

Comment:
I'd rather use my_ allocator here too, so that it's always my_free in the end regardless of is_boot